server/typedb.yml

```diff
@@ -0,0 +1,19 @@
+server:
+    address: 0.0.0.0:1729
+    http_enabled: true
+    http_address: 0.0.0.0:8000
+
+    authentication:
+        token_expiration_seconds: 5000
+
+    encryption:
+        enabled: false
+        # cert: =
+        # cert_key: =
+        # root_ca: =
+
+storage:
+    data: "data"
+
+logging:
+    directory: "logs"
```

Comment:
What about diagnostics config?

Reply:
Can add them, can omit them.

Reply:
If you say so. the first thing I do in a config is disable all diagnostics

Reply:
The deserializer doesn't recognise that 
```
a:
  b:
     c: value
```
is the same as 
```
a.b.c: value
```

I've tried serde_query as well but that didn't work either.

Reply:
^ TODO: 
We only want to support the nested (a:b:c) notation in config files. So I'll have to unflatten Diagnostics struct in core.